source/agora/node/Node.d

```diff
@@ -338,9 +347,67 @@ public class Node : API
         }
     }
 
+    /***************************************************************************
+
+        Returns an instance of a EnrollmentManager
+
+        Params:
+            data_dir = path to the data dirctory
+
+        Returns:
+            the enrollment manager
+
+    ***************************************************************************/
+
+    protected EnrollmentManager getEnrollmentManager (string data_dir)
+    {
+        return new EnrollmentManager(buildPath(data_dir, "validator_set.dat"),
+            config.node.key_pair);
+    }
+
     /// GET: /merkle_path
     public Hash[] getMerklePath (ulong block_height, Hash hash) @safe
     {
         return this.ledger.getMerklePath(block_height, hash);
     }
+
+    /// GET: /get_utxos
+    public UTXOSetValue[Hash] getUTXOs(TxType type, ulong min_amount_value) @safe
+    {
+        UTXOSetValue[Hash] result_utxos;
+        auto utxos = this.utxo_set.getUTXOs(config.node.key_pair.address);
```

Comment:
This is to get an available UTXO list?, but not just the address of the node.
address must also be added by a parameter.

If it's for this node only, need to change the public.